plugin.video.mlbtv/resources/lib/globals.py

```diff
@@ -113,14 +120,14 @@ def createGameIcon(home,away,image_path):
     bg = Image.new('RGB', (500,250), (0,0,0)) 
```

Comment:
This could doesn't get called anymore. Line 67 in main.py hardcodes the game icon for now.